wallet/db.c

```diff
@@ -217,6 +217,16 @@ char *dbmigrations[] = {
     "ALTER TABLE outputs ADD COLUMN spend_height INTEGER REFERENCES blocks(height) ON DELETE SET NULL;",
     /* Create a covering index that covers both fiels */
     "CREATE INDEX output_height_idx ON outputs (confirmation_height, spend_height);",
+    "CREATE TABLE utxoset ("
+    " txid BLOB,"
+    " outnum INT,"
+    " blockheight INT REFERENCES blocks(height) ON DELETE CASCADE,"
+    " spendheight INT REFERENCES blocks(height) ON DELETE SET NULL,"
+    " txindex INT,"
+    " scriptpubkey BLOB,"
+    " satoshis INT,"
```

Comment:
satoshis should be BIGINT.  Doesn't matter for sqlite, but could for other dbs.  Wonder if we get this wrong elsewhere too?

Reply:
Right, presumably we can just replace these without migrations since they map to the same internal datatype in sqlite3. Will change this instance and then go through the others in a new PR.